src/utils/string.ts

```diff
@@ -278,4 +278,34 @@ export const isDataJSON = (data: any) => {
   } catch (error) {
     return false;
   }
+}
+
+/**
+ * Replace snake_case string to camel case
+ * @param str 
+ * @returns 
+ */
+export const toCamel = (str: string) =>{
+  return str.replace(
+    /(?!^)_(.)/g,
+    (_, char) => char.toUpperCase()
+  );
+}
+
+/**
+ * Converts snake_case keys of an object to camel case
+ * @param configObject an object with snake case keys
+ * @returns object with the camel case keys
+ */
+export const gridLayoutConfigMap = (configObject: any) => {
```

Comment:
Is this function specific to `gridLayoutConfig` or is it a general solution to converting from one case to another?

Looks like this is a general function for converting keys in an object from snake case to camel case so the function should be named properly so others can use it too.

Reply:
Yes it's general. I'll fix the name